SofaKernel/modules/SofaBaseMechanics/BarycentricMappers/BarycentricMapperTopologyContainer.h

```diff
@@ -113,20 +144,24 @@ class BarycentricMapperTopologyContainer : public TopologyBarycentricMapper<In,O
     virtual void addPointInElement(const int elementIndex, const SReal* baryCoords)=0;
     virtual void computeDistance(double& d, const Vector3& v)=0;
 
-    void exhaustiveSearch ( defaulttype::Vec3d outPos,
-                            const typename In::VecCoord& in,
-                            const helper::vector<Mat3x3d>& bases,
-                            const helper::vector<Vector3>& centers);
+    void checkDistanceFromElement(unsigned int e,
+                                  const Vector3& outPos,
+                                  const Vector3& inPos,
+                                  NearestParams& nearestParams);
+
+    void computeBasesAndCenters( const typename In::VecCoord& in );
```

Comment:
Thx @EulalieCoevoet for this improvement, could you add a quick doxygen comment explaining what are doing those methods ( use /// before or ///< at end of the line.)
No need for the getter, the name are explicit enough.